prisma/schema.prisma

```diff
@@ -6,6 +6,105 @@ generator client {
 }
 
 datasource db {
-  provider = "postgresql"
-  url      = env("DATABASE_URL")
+  provider          = "postgresql"
+  url               = env("DATABASE_URL")
+  shadowDatabaseUrl = env("DATABASE_SHADOW")
+}
+
+model Admin {
+  id          Int           @id @default(autoincrement())
+  username    String        @unique
+  password    String
+  email       String        @unique
+  createdAt   DateTime      @default(now())
+  updatedAt   DateTime      @updatedAt
+  competition Competition[]
+}
+
+model Competition {
+  id          Int          @id @default(autoincrement())
+  title       String
+  createdAt   DateTime     @default(now())
+  updatedAt   DateTime     @updatedAt
+  Admin       Admin        @relation(fields: [adminId], references: [id])
+  adminId     Int
+  seasons     Season[]
+  competitors Competitor[]
+}
+
+model Competitor {
+  id                 Int         @id @default(autoincrement())
+  name               String
+  nationality        String?
+  email              String
+  competitorImageUrl String?
+  createdAt          DateTime    @default(now())
+  updatedAt          DateTime    @updatedAt
+  Competition        Competition @relation(fields: [competitionId], references: [id])
+  competitionId      Int
+}
+
+model Season {
+  id               Int               @id @default(autoincrement())
+  createdAt        DateTime          @default(now())
+  updatedAt        DateTime          @updatedAt
+  Competition      Competition       @relation(fields: [competitionId], references: [id])
+  competitionId    Int
+  participants     Participant[]
+  teams            Team[]
+  rounds           Round[]
+  positionMappings PositionMapping[]
+}
+
+model Team {
+  id           Int           @id @default(autoincrement())
+  name         String
+  createdAt    DateTime      @default(now())
+  updatedAt    DateTime      @updatedAt
+  Season       Season        @relation(fields: [seasonId], references: [id])
+  seasonId     Int
+  participants Participant[]
+}
+
+model Participant {
+  id         Int         @id @default(autoincrement())
+  createdAt  DateTime    @default(now())
+  updatedAt  DateTime    @updatedAt
+  Season     Season      @relation(fields: [seasonId], references: [id])
+  seasonId   Int
+  Team       Team?       @relation(fields: [teamId], references: [id])
+  teamId     Int?
+  placements Placement[]
+}
```

Comment:
Some are capitalized and some aren't, is there a purpose for these differences?

Reply:
good spot I will change it so that they are all lowercase for uniformity